src/main/java/com/minecolonies/util/SoundUtils.java

```diff
@@ -1,66 +1,117 @@
 package com.minecolonies.util;
 
+import com.minecolonies.entity.EntityCitizen;
+import com.minecolonies.sounds.CitizenSounds;
+import com.minecolonies.sounds.FishermanSounds;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.World;
 
+import java.util.Random;
+
 /**
  * Utilities for playing sounds.
  */
 public final class SoundUtils
 {
-    private static final double HALF_BLOCK_OFFSET  = 0.5D;
-    private static final double STATIC_PITCH_VALUE = 0.9D;
-    private static final double RANDOM_PITCH_VALUE = 0.1D;
+    /**
+     * Get a random between 1 and 100.
+     */
+    private static final int ONE_HUNDRED = 100;
+
+    /**
+     * Standard pitch value.
+     */
+    private static final double PITCH = 0.9D;
+
+    /**
+     * Random object.
+     */
+    private static Random rand = new Random();
+
+    /**
+     * Volume to play at.
+     */
+    private static final double VOLUME = 0.5D;
+
+    /**
+     * in average 1 minute to the next sound which are 20 ticks the second * 60 seconds * 1 minute.
+     */
+    private static final int CHANCE_TO_PLAY_SOUND = 20*60*2;
 
     /**
      * Private constructor to hide the implicit public one
      */
     private SoundUtils()
     {
+        /**
+         * Intentionally left empty.
+         */
     }
 
     /**
-     * {@link #playSound(World, String, double, double, double, float, float)}.
-     *
-     * @param world World to play sound in
-     * @param name  Name of the sound to play
-     * @param x     X-coordinate
-     * @param y     Y-coordinate
-     * @param z     Z-coordinate
+     * Plays a random sound for a certain citizen.
+     * @param worldIn the world to play the sound in.
+     * @param citizen the citizen to play the sound for.
      */
-    /*public static void playSound(@NotNull World world, String name, int x, int y, int z)
+    public static void playRandomSound(World worldIn, EntityCitizen citizen)
     {
-        playSound(world, name,
-          x + HALF_BLOCK_OFFSET, y + HALF_BLOCK_OFFSET, z + HALF_BLOCK_OFFSET,
-          1.0F, (float) (world.rand.nextDouble() * RANDOM_PITCH_VALUE + STATIC_PITCH_VALUE));
-    }*/
+        if(1 >= rand.nextInt(CHANCE_TO_PLAY_SOUND))
+        {
+            String prefix = "";
+
+            if (citizen.getWorkBuilding() != null)
+            {
+                prefix = citizen.getWorkBuilding().getJobName();
+            }
+
+            switch (prefix)
+            {
+                case "Fisherman":
+                    FishermanSounds.playFishermanSound(worldIn, citizen.getPosition(), citizen.isFemale());
+                    break;
+                default:
+                    CitizenSounds.playCitizenSounds(worldIn, citizen.getPosition(), citizen.isFemale());
+                    break;
+            }
+        }
+    }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)